misocho/questioner
# #163190909 Adds user down-vote, change up-vote button

#### What does this PR do?
* Adds the downvote icon
* Changes the upvote icon
#### Description of Task to be completed?
* Add downvote icon
* Change downvote icon
#### How should this be manually tested?
* Clone the GitHub [repo](https://github.com/misocho/questioner.git)
* Navigate to project directory
* Create a branch and pull from ft-user-downvote-163190909
* Pull from ft-user-downvote-163190909
* Open questions.html file with your browser
* You will be able to see the up-vote and down-vote buttons
### What are the relevant pivotal tracker stories?
#163190909
pt-link - https://www.pivotaltracker.com/n/projects/2235306
#### Screenshots
![image](https://user-images.githubusercontent.com/29132159/51117919-23ca6e80-1820-11e9-80b4-afd262989936.png)
